openreview/venue/process/submission_commitments_preprocess.py

```diff
@@ -0,0 +1,17 @@
+def process(client, edit, invitation):
+
+    paper_link = edit.note.content['paper_link']['value']
+    paper_forum = paper_link.split('=')[-1]
+
+    try:
+        client_v1=openreview.Client(baseurl=openreview.tools.get_base_urls(client)[0], token=client.token)
+        arr_submission = client_v1.get_note(paper_forum)
+    except openreview.OpenReviewException as e:
+        raise openreview.OpenReviewException('Provided paper link does not correspond to a submission in OpenReview')
+
+    valid_prefixes = ['aclweb.org/ACL/ARR/2021', 'aclweb.org/ACL/ARR/2022', 'aclweb.org/ACL/ARR/2023', 'aclweb.org/ACL/ARR/2024']
```

Comment:
what if we have one single prefix `aclweb.org/ACL/ARR`?

Reply:
Yes! Not sure why we were using the year here...I don't see any other venue ids that would start with this.